content/pt-br/docs/reference/glossary/csi.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Interface de Armazenamento de Contêiner
+id: csi
+date: 2018-06-25
+full_link: /pt-br/docs/concepts/storage/volumes/#csi
+short_description: >
+ A Interface de Armazenamento de Contêiner (Container Storage Interface - CSI) define um padrão de interface para expor sistemas de armazenamento a contêineres.
+
+aka: 
+tags:
+- storage 
+---
+ A Interface de Armazenamento de Contêiner (Container Storage Interface - CSI) define um padrão de interface para expor sistemas de armazenamento a contêineres.
```

Comment:
Suggestion: stylize "Container Storage Interface" as italic and use ", ou" instead of a dash to separate it from the acronym.